master/static/migrations/20240212130408_proto-time-helper.tx.down.sql

```diff
@@ -0,0 +1 @@
+DROP FUNCTION IF EXISTS public.proto_time;
```

Comment:
doesn't this need to replace the old version? can you do that so the change is easier to review?

Reply:
like, this file should be
```
CREATE OR REPLACE FUNCTION public.proto_time(ts timestamptz)
    RETURNS json
    LANGUAGE sql IMMUTABLE
    RETURNS NULL ON NULL INPUT
AS $$
    SELECT json_build_object(
        -- Seconds since epoch
        'seconds',  floor(extract(epoch FROM ts))::bigint, 
        -- Fractional part in nanos since epoch
        'nanos',    ((extract(epoch FROM ts) - floor(extract(EPOCH FROM ts))) * 1000000000)::int
    )
$$;
```

Reply:
Yes, you're right I'll fix it